doc/whats_new/v1.1.rst

```diff
@@ -508,8 +508,8 @@ Changelog
 
 - |Fix| :class: `pipeline.FeatureUnion` does not validate hyper-parameters in
   `__init__`. Validation is now handled in `.fit()` and `.fit_transform()`.
-  :pr:`21954` by :user:`iofall <iofall>` and :user:`Arisa Y. <arisayosh>`.
-  :pr:`21888` by :user:`iofall <iofall>` and :user:`Arisa Y. <arisayosh>`.
+  :pr:`21954` and :pr:`21888` by :user:`iofall <iofall>` and
```

Comment:
@glemaitre, I realize after seeing this change that I had accidentally left the line for `:pr:21888`. Sorry about that. This line should only be:

```:pr:`21954` by :user:`iofall <iofall>` and :user:`Arisa Y. <arisayosh>`.```

How should we fix this? Should I add the commit on this PR or create a new PR?

Reply:
Oh I see no worries. I will make a small doc fix in `main`.

Reply:
FYI, it will be fixed in https://github.com/scikit-learn/scikit-learn/pull/22347

Reply:
Thank you!